news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+
+Germany finally took their first point on [Hold Your Colour](https://osu.ppy.sh/beatmapsets/702272#osu/1485947), and while they lost two more maps to bring the score to 6–1, ::{ flag=DE }:: [Tim Kackner](https://osu.ppy.sh/users/14385814) put up a Hard Rock FC on [Eiyuu no Tsurugi](https://osu.ppy.sh/beatmapsets/1864097#osu/3833517) to win it, followed by a ::{ flag=DE }:: [criller](https://osu.ppy.sh/users/8116659) and ::{ flag=DE }:: [WhiteCat](https://osu.ppy.sh/users/4504101) masterclass on [Mondaymania](https://osu.ppy.sh/beatmapsets/2284699#osu/4872865) to put up their third point of the match.
+
+Australia, though, would not be denied. It was a double FC from ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) and ::{ flag=AU }:: [Zyntex](https://osu.ppy.sh/users/11806306) on [roots](https://osu.ppy.sh/beatmapsets/2139515#osu/4502914) to seal the deal, a 7–3 win for the Aussies, sending them onward to the Grand Finals week, a top 3 finish secured for a second year in a row and a date set with familiar nemeses the United States.
+
+It was the emergence of the ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) show in the late stages, where skill cap trumps all, as he top scored 5 of 9 maps — he would finish the week with an eye-popping 1.782 performance score, the highest of the entire field. Germany, for their part, had done their job and done it well — they had finished top 4 for the 7th time straight and top 6 for a remarkable 11th year in a row, a legacy of performance few other teams could ever hope to match.
+
+### ::{ flag=KR }:: South Korea vs. ::{ flag=US }:: United States
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116363241)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2315417594)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/-H-rOKG-2C4?start=645&end=948" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Throughout this world cup, and throughout the finals weekend, there has been one matchup on people's minds — the anticipated Grand Finals matchup, and the matchup from Grand Finals in 2023, South Korea vs. the United States. With how close last year's Grand Finals was and how much both teams improved in the year since, expectations for a close, high-level match were sky-high, and this winners' Finals delivered.
```

Comment:
```suggestion
Throughout this world cup, and throughout the Finals weekend, there has been one matchup on people's minds — the anticipated Grand Finals matchup, and the matchup from Grand Finals in 2023, South Korea vs. the United States. With how close last year's Grand Finals was and how much both teams improved in the year since, expectations for a close, high-level match were sky-high, and this winners' Finals delivered.
```